### ISSUE TITLE
Uncaught RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by laukstein** [sampotts/plyr#1621]
Exception happens on load on latest Plyr 3.5.6, on Chrome 78, Android 8 device.

```
src/js/utils/is.js
const isNumber = input => getConstructor(input) === Number && !Number.isNaN(input);
src/js/utils/time.js
if (!is.number(time)) { ...
```

![grabilla em9460](https://user-images.githubusercontent.com/183966/69708671-b6a48c80-1104-11ea-98e1-c625d3772a50.png)
![grabilla em4420](https://user-images.githubusercontent.com/183966/69708683-ba381380-1104-11ea-9361-dd0f8abca29c.png)

**Comment by laukstein** [comment]
Same error also in `formatTime(null, displayHours, inverted)`
![grabilla g16832](https://user-images.githubusercontent.com/183966/71554036-9842f100-2a22-11ea-8829-a43680aed2a7.png)

**Comment by AMC12010** [comment]
Try Using Chrome.... @laukstein

**Comment by laukstein** [comment]
@AMC12010 see my above message:

> Exception happens on load on latest Plyr 3.5.6, on **Chrome 78**, Android 8 device.

**Comment by sampotts** [comment]
I can't replicate this on iOS 13.3 - any help appreciated on how I can replicate so I can fix. I'm a little lost.

**Comment by laukstein** [comment]
@sampotts maybe try on BrowserStack.
Here I just create you pull request for this issue fix https://github.com/sampotts/plyr/pull/1655 (I haven't tested it, but I think it is the right fix).

The issue was that formatTime was running itself creating infinite loop. `null` is considered as a value and wouldn't use arguments placeholder.
See your issue
![grabilla g22204](https://user-images.githubusercontent.com/183966/72872061-111d2c00-3cf5-11ea-84c4-a49fabe023f9.png)

See how it should be
![grabilla g20272](https://user-images.githubusercontent.com/183966/72872080-1c705780-3cf5-11ea-9e5d-b56866083045.png)

**Comment by laukstein** [comment]
BTW please check if you don't have this bug also  in other places.

**Comment by sampotts** [comment]
Fixed in v3.5.7. Feel free to check other places if you have time... this is an _open source_, not a commercial project.
